plenty_api/api.py

```diff
@@ -145,7 +145,7 @@ class PlentyApi():
     """
     def __init__(self, base_url: str, use_keyring: bool = True,
                  data_format: str = 'json', debug: bool = False,
-                 username: str = '', password: str = ''):
+                 username: str = '', password: str = '', use_gpg: bool = True):
```

Comment:
I would like to get a view of the use case here. Does this mean that you store the plain password within a file/configuration, read it out within the user application, and put it into the init? If that is the case, what exactly didn't you like about the gpg file/ keyring method? Is it the authentication? I simply ask because I don't want to encourage users of the API interface to use unsafe methods for storing their passwords. Especially, because an attacker can do quite a bit of financial damage when he gets access to the REST API account of a plentymarkets system.

Reply:
I think it should be up to the user of the library to make sure whatever way to store credentials is safe. Encrypted files and keyrings are good ways to do this, but you might also get them from a database, from some window or from the terminal (which you already implemented). This change allows the program implementing your library to provide a password from an alternative source without any workarounds.